examples/example-jack-new/build.rs

```diff
@@ -0,0 +1,105 @@
+use faust_build::faust_arg::{FaustArg, FaustArgs2CommandArgs};
+use faust_json::deserialize::FaustJson;
+use proc_macro2::TokenStream;
+use quote::{format_ident, quote};
+use std::{
+    fs::{self, File},
+    io::BufReader,
+    path::{Path, PathBuf},
+    process::Command,
+};
+use syn::{parse_str, Ident};
+
+fn generate_dsp_code(struct_name: &str, dsp_path: &str) -> proc_macro2::TokenStream {
+    let args: Vec<FaustArg> = vec![
+        FaustArg::default_lang(),
+        FaustArg::default_timeout(),
+        FaustArg::DebugWarnings,
+        FaustArg::StructName(struct_name.to_string()),
+        FaustArg::Json(),
+        FaustArg::DspPath(PathBuf::from(dsp_path)),
+    ];
+
+    let mut faust = Command::new("faust");
+    faust.args(args.to_command_args());
```

Comment:
Why do we need to manually convert it here? Could we change faust.args to something that takes `Into<Vec<FaustArg>>` ?